app/assets/javascripts/import_button.js.coffee.erb

```diff
@@ -16,3 +16,6 @@
 $ ->
   import_button_html = '<div class="input-group-btn"><button type="submit" name="media_object[import_bib_record]" class="btn btn-success" value="yes">Import</button></div>'
   $('div.import-button').append(import_button_html)
+  $('div.import-button button').click ->
+    warning = '<%= I18n.t(:bib_import_warning) %>';
```

Comment:
Yeah, I'm not crazy about this.

One thing I might want to do is have an init method that gets called in the view that sets the warning. E.g., within this JS file, define a function set_bib_import_warning_message(msg) that sets a variable and has no dependencies on I18n, then in the view file that needs this, call the function with set_bib_import_warning_message('<%= I18n.t(:bib_import_warning) %>').